pom.xml

```diff
@@ -19,27 +20,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.66</version>
+        <version>4.75</version>
         <relativePath />
     </parent>
 
     <artifactId>fortify</artifactId>
-    <version>23.1.40-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
 
     <properties>
-        <gitHubRepo>jenkinsci/fortify-plugin</gitHubRepo>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <revision>23.1.40</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```

Comment:
As per latest recommendations in the archetype: https://github.com/jenkinsci/archetypes